js/script.js

```diff
@@ -1177,6 +1177,24 @@ var validCombos = {
 
     "linux,C++,X86,XNNPACK":
         "Follow build instructions from <a href='https://aka.ms/build-ort-xnnpack' target='_blank'>here</a>",
+
+    "linux,Python,ARM64,CANN":
+        "pip install onnxruntime-cann <br/>Refer to <a href='http://www.onnxruntime.ai/docs/execution-providers/CANN-ExecutionProvider.html#requirements' target='_blank'>docs</a> for requirements.",
```

Comment:
are you guys planning on publishing an onnxruntime-cann package to pypi?

Reply:
Yes, plan to publish it with version 1.13.0 of onnxruntime.

Reply:
please note there is a rel-1.3.1 branch. track that one instead of 1.3.0
thanks